allzpark/control.py

```diff
@@ -1091,8 +1091,11 @@ def _list_apps(self, profile):
         def _try_finding_latest_app(req_str):
             req_str = req_str.strip("~")
             req = rez.PackageRequest(req_str)
+            package_filter = self._package_filter()
```

Comment:
This line is duplicated I think, the package filter object has already been presented at line 1089. ☝🏼